repos/spack_repo/builtin/packages/cargo_c/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack_repo.builtin.build_systems.cargo import CargoPackage
+
+from spack.package import *
+
+
+class CargoC(CargoPackage):
+    """A package to build and install C-compatible libraries"""
+
+    git = "https://github.com/lu-zero/cargo-c.git"
+    url = "https://github.com/lu-zero/cargo-c/archive/refs/tags/v0.10.18.tar.gz"
```

Comment:
```suggestion
    homepage = "https://github.com/lu-zero/cargo-c"
    url = "https://github.com/lu-zero/cargo-c/archive/refs/tags/v0.10.18.tar.gz"
    git = "https://github.com/lu-zero/cargo-c.git"
```

Let's include the homepage and order the properties like other packages for readability.